tests/robot-tests/tests/general_public/table_tool_absence_by_characteristic.robot

```diff
@@ -240,15 +240,15 @@ Validate rows after reordering
 
 User generates a permanent link
     [Tags]   HappyPath
-    user waits until page contains button  Share your table 
-    user clicks button  Share your table
+    user waits until page contains button  Generate shareable link 
+    user clicks button  Generate shareable link
     user waits until page contains element  xpath://a[text()="View share link"]  60
     user checks generated permalink is valid
 
 User validates permanent link works correctly
     [Tags]   HappyPath
     user clicks link   View share link
-    select window    NEW
+    switch window   locator=NEW
```

Comment:
Can use the new `user selects newly opened window` keyword you've created here.

Reply:
Noice thanks